tests/test_visitors/test_ast/test_classes/test_wrong_empty_lines_count.py

```diff
@@ -30,6 +30,24 @@ def wrong_method(self):
         baz()
 """
 
+file_with_few_class = """
```

Comment:
Let's add more cases:
- classes with lots of empty lines and class-level attributes, but no functions
- expressions with no functions, but with lots of empty lines, like function calls
- module-level empty lines
- etc